packages/cf-component-checkbox/src/CheckboxGroup.js

```diff
@@ -27,7 +38,7 @@ class CheckboxGroup extends React.Component {
 
   render() {
     return (
-      <div className="cf-checkbox__group">
+      <div>
         {this.props.options.map(option => {
```

Comment:
I would probably prefer this API (and utilize React.Children.map +  React.cloneElement)

```jsx
<CheckboxGroup>
  <Checkbox />
  <Checkbox />
</CheckboxGroup>
```

instead of 

```jsx
<CheckboxGroup options={...} />
```

so it follows the `ButtonGroup` and `Button` example. The later seems more as a builder pattern (more restrictive). However, since you are just keeping the existing solution here, let's revisit this another day.